platform/feature-api/api/pod-api/src/spec.ts

```diff
@@ -151,7 +153,7 @@ export class PodSpec {
                         fc.asyncProperty(pathGen, async (path) => {
                             await skipIfExists(path);
 
-                            await assert.isRejected(polyOut.readFile(path, { encoding: "utf-8" }));
+                            await assert.isRejected(polyOut.readFile(path));
```

Comment:
What's the deal with the encoding?

Reply:
We don't support `readFile` with options (like `encoding`) - because we don't need it. So no need to test something we don't need... I think we can probably safely remove it, I can hardly think of a use case, given that the file system is essentially virtual.

Reply:
It might be there when readFile was mostly delegated to fs and fs polyfills, I think. But you're right, it makes sense to just remove it right now.